### PR TITLE
Add readme VC description

### PR DESCRIPTION
**Pull request by WojciechBarczynski**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,72 @@
...
+```console
+curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs | sh
+```
+
+#### FFmpeg installation
+
+##### Ubuntu
+
+```console
+sudo apt-get install libavcodec-dev libavformat-dev libavutil-dev
```
[the remark this answers is not in the document]
Comment (WojciechBarczynski) [reply README.md]: I removed those commands from readme

File: README.md [hunk 2]
```diff
@@ -1,2 +1,72 @@
...
+brew install ffmpeg
+```
+
+### Running
+
+For better performance, build examples with the [release compilation profile](https://doc.rust-lang.org/book/ch14-01-release-profiles.html):
+
+```console
+cargo run --release --example <example_name>
+```
```
[the remark this answers is not in the document]
Comment (WojciechBarczynski) [reply README.md]: This was nested inside `Examples`
```markdown
## Examples
...
### Running
```
but I simplified it anyway.

File: README.md [hunk 3]
```diff
@@ -1,2 +1,44 @@
...
+
+## Supported platforms
+
+Currently, we support Linux and MacOS.
```
Comment (wkozyra95) [inline README.md:38]:
```suggestion
Linux and MacOS
```
Too much currently

File: README.md [hunk 4]
```diff
@@ -1,2 +1,44 @@
...
+Application for real-time video processing/transforming/composing, providing simple, language-agnostic API for live video rendering.
+
+VideoCompositor targets real-time use cases, like video conferencing, live-streaming, or broadcasting (e.g. with [WebRTC](https://en.wikipedia.org/wiki/WebRTC) / [HLS](https://en.wikipedia.org/wiki/HTTP_Live_Streaming) / [RTMP](https://en.wikipedia.org/wiki/Real-Time_Messaging_Protocol)).
+
+## Features
+
+VideoCompositor receives inputs and sends outputs streams via [RTP](https://en.wikipedia.org/wiki/Real-time_Transport_Protocol).
+HTTP API is used to define how inputs should be transformed and combined to produce desired outputs.
+
+Currently, we want VideoCompositor to have four types of transformations, that you can combine together:
```
Comment (wkozyra95) [inline README.md:12]:
```suggestion
For the initial release, we want VideoCompositor to support those four types of transformations, that you can combine together:
```

**Review by wkozyra95 — APPROVED** [review]

**Pull request merged**